release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1305,6 +1305,21 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-15-management-console-bug-fixes"]
 ==== Management Console
 
+* Previously, the *Expand PVC* modal assumed the existing PVC had a `spec.resources.requests.storage` value that includes a unit. As a result, when the *Expand PVC* modal was used to expand a PVC that had a `requests.storage` value without a unit, the console would display an incorrect value in the modal. With this update, the console was updated to handle storage values with and without a unit. (link:https://issues.redhat.com/browse/OCPBUGS-27909[*OCPBUGS-27909*])
+
+* Previously, the *Expand PVC* modal assumed the existing PVC had a `spec.resources.requests.storage` value that includes a unit. As a result, when the *Expand PVC* modal was used to expand a PVC that had a `requests.storage` value without a unit, the console would display an incorrect value in the modal. With this update, the console was updated to handle storage values with and without a unit. (link:https://issues.redhat.com/browse/OCPBUGS-27177[*OCPBUGS-27177*])
+
+* Previously, the console check to determine if a file is binary was not robust enough. As a result, XML files were misidentified as binary and not displaying in the console. With this update, an additional check was added to more precisely check if a file is binary. (link:https://issues.redhat.com/browse/OCPBUGS-26591[*OCPBUGS-26591*])
+
+* Previously, the *Node Overview* page failed to render when a `MachineHealthCheck` without `spec.unhealthyConditions` existed on a cluster. With this update, the *Node Overview* page was updated to allow for `MachineHealthChecks` without `spec.unhealthyConditions`. Now, the *Node Overview* page renders even if `MachineHealthChecks` without `spec.unhealthyConditions` are present on the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-25140[*OCPBUGS-25140*])
+
+* Previously, the console was not up to date with the newest matchers key for alert notification receivers, and the alert manager receivers created by the console utilized the older match key. With this update, the console uses matchers instead, and converts any existing match instances to matchers when modifying an existing alert manager receiver. (link:https://issues.redhat.com/browse/OCPBUGS-23248[*OCPBUGS-23248*])
+
+* Previously, impersonation access was incorrectly applied. With this update, the console correctly applies impersonation access. (link:https://issues.redhat.com/browse/OCPBUGS-23125[*OCPBUGS-23125*])
+
+* Previously, when the Advanced Cluster Management for Kubernetes (ACM) and multicluster engine for Kubernetes (MCE) Operators are installed and their plugins are enabled, the yaml code Monaco editor failed to load.
+With this update, optional resource chaining was added to the to prevent a failed resource call, and the yaml editor no longer fails to load when the ACM and MCE Operators are installed and their plugins enabled. (link:https://issues.redhat.com/browse/OCPBUGS-22778[*OCPBUGS-22778*])
+
```

Comment:
Should "resource chaining was added to the to prevent a failed resource call" be "resource chaining was added to prevent a failed resource call"? Or "resource chaining was added to the <MISSING WORD> to prevent a failed resource call..."

Seems like either there's an extra "to" and unnecessary "the," or there is a word missing after "the."

Reply:
Changed to make sense, thanks!